src/components/UserPathLog/Login.js

```diff
@@ -47,24 +46,46 @@ export default function Login() {
   return (
     <React.Fragment>
       <div className="login">
-        <h1>Login Page</h1>
-        <label> user name </label>
+        <Logo />
+
+        <h1>Welcome</h1>
+
         <input
+          className="input-field"
           type="text"
           placeholder="username"
           onChange={(e) => setUsername(e.target.value)}
           value={username}
         />
-        <label> password </label>
+
         <input
+          className="input-field"
           type="password"
           onChange={(e) => setPassword(e.target.value)}
           value={password}
         />
-        <button onClick={login}> Login </button>
+        <Button
+          onClick={login}
+          type="button"
+          buttonStyle="btn--primary--solid"
+          buttonSize="btn--medium"
+        >
+          Login
+        </Button>
+      </div>
+      <div className="links-container">
+        <Link to="/register">
```

Comment:
<Link> is something we're using from the react-router.  It's not an onClick and it won't mess with any of the rendering of you Button component, you can just wrap your Button component in the <Link> component to give it the functionality you're looking for!